### PR TITLE
Add how-to document on ARM FVP

### PR DESCRIPTION
**Pull request by wuxiaoqiang12**

### CODE REVIEW

File: Documents/004-Test_Minos_on_ARM_FVP.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+   make ARCH=arm64 CROSS_COMPILE=aarch64-linux-gnu- -j8 Image
+   ```
+
+## 4. Compile ARM Trusted Firmware
+
+   ```shell script
+   cd ~/minos-workspace
+   git clone https://github.com/ARM-software/arm-trusted-firmware.git
+   cd arm-trusted-firmware
+   make CROSS_COMPILE=aarch64-linux-gnu- PLAT=fvp RESET_TO_BL31=1 ARM_LINUX_KERNEL_AS_BL33=1 PRELOADED_BL33_BASE=0xc0000000 ARM_PRELOADED_DTB_BASE=0x83e00000
```
Comment (lemin9538) [inline Documents/004-Test_Minos_on_ARM_FVP.md]: PRELOADED_BL33_BASE=**### 0xc0000000** ARM_PRELOADED_DTB_BASE=**### 0x83e00000**

These two address is for old version of minos. need to update to the latest one

Comment (wuxiaoqiang12) [reply Documents/004-Test_Minos_on_ARM_FVP.md]: Done

Comment (lemin9538) [inline Documents/004-Test_Minos_on_ARM_FVP.md]: PRELOADED_BL33_BASE=### 0xc0000000 ARM_PRELOADED_DTB_BASE=### 0x83e00000

These two address is for old version of minos. need to update to the latest one 0xc0008000 and 0xc3e0000

File: Documents/004-Test_Minos_on_ARM_FVP.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  -C cluster0.cpu3.RVBAR=0x04020000 \
+  -C bp.hostbridge.userNetPorts="8023=22" \
+  -C bp.hostbridge.userNetworking=true \
+  -C bp.dram_size=8 \
+  -C bp.smsc_91c111.enabled=true \
+  -C bp.virtioblockdevice.image_path=/home/{whoami}/minos-workspace/arm-fvp/sd.img \
+  --data cluster0.cpu0=/home/{whoami}/minos-workspace/arm-fvp/bl31.bin@0x04020000 \
+  --data cluster0.cpu0=/home/{whoami}/minos-workspace/arm-fvp/Image@0x80080000 \
+  --data cluster0.cpu0=/home/{whoami}/minos-workspace/arm-fvp/minos.bin@0xc0008000 \
+  --data cluster0.cpu0=/home/{whoami}/minos-workspace/arm-fvp/fdt.dtb@0xc3e00000
```
[the remark this answers is not in the document]
Comment (wuxiaoqiang12) [reply Documents/004-Test_Minos_on_ARM_FVP.md]: Done. Please feel free to give me more comments to help me improve my pull request if you have more advices. Thanks a lot.

[the remark this answers is not in the document]
Comment (lemin9538) [reply Documents/004-Test_Minos_on_ARM_FVP.md]: Good, the guide is very detailed.

**Comment by lemin9538** [comment]
One more thing I forgot, I didn't notice that you add 4 commits for this topic, usually you only need to submit one commit for one topic. you can use
# git add .
# git commit --amend

if you add new modification based previous commit, you can use "git commit --amend",

I will mege your pull request manually.

**Comment by wuxiaoqiang12** [comment]
Ok. Thank you for your help.

**Comment by lemin9538** [comment]
> Ok. Thank you for your help.

Another thing is that:
when you do commit please add **-s** parameter this will add Signed-off-by field like below:

# git commit -s

commit 735ee698542926aaad0cdbd05c5884a80bb09032 (HEAD -> master)
Author: wuxiaoqiang12 <xiaoqiangwu.hit@gmail.com>
Date:   Mon Mar 16 22:30:20 2020 -0400

    add how-to document for ARM FVP
    
    Signed-off-by: wuxiaoqiang12 <xiaoqiangwu.hit@gmail.com>

**Comment by wuxiaoqiang12** [comment]
Oh. I see. I learn lots of things from you. Thanks.

**Pull request closed without merging**